src/3d/qgsdirectionallightsettings.h

```diff
@@ -0,0 +1,66 @@
+#ifndef QGSDIRECTIONALLIGHTSETTINGS_H
+#define QGSDIRECTIONALLIGHTSETTINGS_H
```

Comment:
would be good to move the header guards below the copyright comment - just for consistency